pkg/sync/syncer.go

```
@@ -852,99 +852,9 @@
-				)
-				if err != nil {
-					return fmt.Errorf("error adding edge to graph: %w", err)
-				}
-			}
-		}
-		return nil
-	}
-
+	fmt.Printf("%v\n", entitlementGraph)
 	if entitlementGraph.Loaded {
+		fmt.Printf("getting cycles1\n")
```

golangci-lint (1.62.2)

use of `fmt.Printf` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$`

(forbidigo)